deep_speech_2/model_utils/model.py

```diff
@@ -79,7 +80,7 @@ def train(self,
         """
         # prepare model output directory
         if not os.path.exists(output_model_dir):
-            os.mkdir(output_model_dir)
+            mkpath(output_model_dir)
```

Comment:
Use `os.makedirs(output_model_dir)` ?

Reply:
https://stackoverflow.com/questions/600268/mkdir-p-functionality-in-python